tests/phpunit/tests/post/truncatePostSlug.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * @group post
+ *
+ * @covers ::_truncate_post_slug
+ */
+class Tests_Post_TruncatePostSlug extends WP_UnitTestCase {
+
+	/**
+	 * Tests that _truncate_post_slug() correctly truncates slugs.
+	 *
+	 * @ticket 56868
+	 *
+	 * @dataProvider data_truncate_post_slug_should_truncate
+	 *
+	 * @param string $slug     The slug to truncate.
+	 * @param int    $length   Max length of the slug.
+	 * @param string $expected The expected truncated slug.
+	 * @param string $message  Test feedback message.
+	 */
+	public function test_truncate_post_slug_should_truncate( $slug, $length, $expected, $message ) {
+		$this->assertSame( $expected, _truncate_post_slug( $slug, $length ), $message );
+	}
+
+	/**
+	 * Data provider for test_truncate_post_slug_should_truncate().
+	 *
+	 * @return array[]
+	 */
+	public function data_truncate_post_slug_should_truncate() {
+		return array(
+			'a slug that is too long'                      => array(
```

Comment:
```suggestion
			'a slug that is too long' => array(
```

Don't think we needs to align the arrows.